src/routes/ChatList.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+import { HeaderContains, HeaderText } from '../components/layout/CommonStyle';
+import UnderNavBar from '../components/layout/UnderNavBar';
+import Messages from '../components/ChatList/Chattings';
+import message from '../data/message.json';
+import { BsChatDots } from 'react-icons/bs';
+
+const ChatList = () => {
+  return (
+    <ChatListContainer>
+      <HeaderContains>
+        <BsChatDots size={20} />
+        <HeaderText>Message</HeaderText>
+      </HeaderContains>
+      <Content>
+        {message.map((userMessage) => (
+          <Messages
+            key={userMessage.userId}
+            userId={userMessage.userId}
+            userName={userMessage.userName}
+            message={userMessage.messages[userMessage.messages.length - 1].text}
```

Comment:
저는 **개인적으로** 배열의 이름은 복수형으로 짓고, 콜백함수의 props는 단수형으로 작성하는 편이 조금 더 직관적인 것 같아서 이런 식으로 사용합니다...!

```suggestion
        {messages.map((message) => (
          <Messages
            key={message.userId}
            userId={message.userId}
            userName={message.userName}
            message={message.messages[userMessage.messages.length - 1].text}
```
  
아니면 아래처럼 destructuring을 사용하기도 합니다
```suggestion
        {message.map(({ userId, userName, messages }) => (
          <Messages
            key={userId}
            userId={userId}
            userName={userName}
            message={messages[userMessage.messages.length - 1].text}
```

Reply:
네.. 코드가 길어지면서 저도 이번 과제하면서 네이밍이 참 어렵고 맘에 안들었는데 ..... 팁 감사합니다 💗 반영해서 수정해볼게요!!!!